files/ldap/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ version: '3'
 services:
   openldap:
     restart: "always"
-    image: osixia/openldap:1.2.4
+    image: osixia/openldap:1.5.0
```

Comment:
Let's not upgrade here. It's a production deployment and we don't want to handle that as part of testing the staging environment.

We will need to upgrade it separately.

Reply:
Sure. I will fix it.